File: BMO/BMO/Mun/frmMenu.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace BMO
{
    public partial class frmMenu : Form 
    {
       
        public frmMenu()
        {
            InitializeComponent();
        }

        private void BtnAtbash_Click(object sender, EventArgs e)
        {
            Mun.OpcionesMenu ob = new Mun.OpcionesMenu();
            this.Close();
            ob.OpenAtbash();
        }

        private void BtnCesar_Click(object sender, EventArgs e)
        {
            Mun.OpcionesMenu ok = new Mun.OpcionesMenu();
            this.Close();
            ok.ApertoCesar();
        }

        private void BtnSalir_Click(object sender, EventArgs e)
        {
            Application.Exit();
        }
    }
}

File: BMO2/BMO2/Mun/frmMenu.vb
Public Class frmMenu
    Private Sub BtnAtbash_Click(sender As Object, e As EventArgs) Handles btnAtbash.Click
        Dim OpenAtbash As New OpcionesMenu
        OpenAtbash.Open()
        Me.Close()
    End Sub

    Private Sub BtnCesar_Click(sender As Object, e As EventArgs) Handles btnCesar.Click
        Dim ApertoCesar As New OpcionesMenu
        ApertoCesar.Aperto()
        Me.Close()
    End Sub

    Private Sub BtnSalir_Click(sender As Object, e As EventArgs) Handles btnSalir.Click
        Application.Exit()
    End Sub
End Class